DependencyInjection/GraphQLiteCompilerPass.php

```diff
@@ -479,34 +479,18 @@ private function getCodeCache(): ClassBoundCacheContractInterface
      * Returns the array of globbed classes.
      * Only instantiable classes are returned.
      *
-     * @return array<string,ReflectionClass<object>> Key: fully qualified class name
+     * @param string $namespace
+     * @return array<class-string, ReflectionClass<object>>|Generator Key: fully qualified class name
      */
-    private function getClassList(string $namespace, int $globTtl = 2, bool $recursive = true): array
+    private function getClassList(string $namespace): array|Generator
```

Comment:
This is always a `Generator`


```suggestion
    private function getClassList(string $namespace): Generator
```

Also add a phpstan-compatible `@return` annotation https://phpstan.org/blog/generics-in-php-using-phpdocs#type-safe-iterators-and-generators:

```php
 * @return Generator<string, ReflectionClass, void, void>
```